modules/persistent-storage-csi-snapshots-restore.adoc

```diff
@@ -5,14 +5,15 @@
 [id="persistent-storage-csi-snapshots-restore_{context}"]
 = Restoring a volume snapshot
 
-After your `VolumeSnapshot` object is bound, you can use that object to provision a new volume that is pre-populated with data from the snapshot.
+The volume snapshot content object can be used to restore the existing volume to a previous state.
 
-The volume snapshot content object is used to restore the existing volume to a previous state.
+After your `VolumeSnapshot` object is bound and the `readyToUse` value is set to `true`, you can use that object to provision a new volume that is pre-populated with data from the snapshot.
 
 .Prerequisites
 * Logged in to a running {product-title} cluster.
-* A persistent volume claim (PVC) created using a Container Storage Interface (CSI) driver that supports volume snapshots.
+* A PVC created using a Container Storage Interface (CSI) driver that supports volume snapshots.
```

Comment:
This reference to PVC is the first in the module, so I would suggest keeping it as it was. (Unless something changed :) )

Reply:
Fixed, thanks!